### PR TITLE
Added the mountISOPmem function that mounts /dev/pmem0 onto a directory.

### PR DESCRIPTION
**Pull request by louismurerwa**
Signed-off-by: Louis Murerwa <louis.b.murerwa.21@dartmouth.edu>

### CODE REVIEW

File: pkg/mountkexec/mountkexec.go [hunk 1]
```diff
@@ -39,6 +39,19 @@ func MountISO(isoPath, mountDir string) error {
 	return nil
 }
 
+// MountISOPmem creates a directory and mounts the pmem block device on the directory
+func MountISOPmem(Pmem, mountDir string) error {
```
Comment (rminnich) [inline pkg/mountkexec/mountkexec.go]: change the variable Pmem to pmem

File: pkg/mountkexec/mountkexec.go [hunk 2]
```diff
@@ -39,6 +39,19 @@ func MountISO(isoPath, mountDir string) error {
...
+	if err := os.MkdirAll(mountDir, 777); err != nil {
+		return fmt.Errorf("error making mount directory:%v", err)
+	}
+	var flags uintptr
+	flags |= unix.MS_RDONLY
+	if err := mount.Mount(Pmem, mountDir, "iso9660", "", flags); err != nil {
```
Comment (rminnich) [inline pkg/mountkexec/mountkexec.go]: you can just replace the flags parameter in Mount with unix.MS_RDONLY and remove lines 47 and 48

File: pkg/mountkexec/mountkexec.go [hunk 3]
```diff
@@ -39,6 +39,19 @@ func MountISO(isoPath, mountDir string) error {
...
+		return fmt.Errorf("error mounting ISO:%v", err)
```
Comment (rminnich) [inline pkg/mountkexec/mountkexec.go]: Might as well tell me more: error mounting %v on %v: %v", pmem, mountDir, err)

File: pkg/mountkexec/mountkexec.go (hunk 1 above)
Comment (louismurerwa) [reply pkg/mountkexec/mountkexec.go]: Done

File: pkg/mountkexec/mountkexec.go (hunk 3 above)
Comment (louismurerwa) [reply pkg/mountkexec/mountkexec.go]: Done.

File: pkg/mountkexec/mountkexec.go (hunk 2 above)
Comment (louismurerwa) [reply pkg/mountkexec/mountkexec.go]: Done

**Pull request merged**